internal/utils/csi.go

```diff
@@ -28,11 +28,13 @@ const (
 	csiEndpoint       = "unix://" + SocketDir + "/csi.sock"
 	csiAddonsEndpoint = "unix://" + SocketDir + "/csi-addons.sock.sock"
 
-	kmsConfigVolumeName      = "ceph-csi-kms-config"
-	registrationVolumeName   = "registration-dir"
-	pluginDirVolumeName      = "plugin-dir"
-	podsMountDirVolumeName   = "pods-mount-dir"
-	pluginMountDirVolumeName = "plugin-mount-dir"
+	kmsConfigVolumeName              = "ceph-csi-kms-config"
+	registrationVolumeName           = "registration-dir"
+	pluginDirVolumeName              = "plugin-dir"
+	podsMountDirVolumeName           = "pods-mount-dir"
+	pluginMountDirVolumeName         = "plugin-mount-dir"
+	csiLogDirectoryVolumeName        = "csi-log"
```

Comment:
please make the var name match the exiting conventions:
So in this case, if the volume name is `csi-log` the const name should be `csiLogVolumeName`?

In addition, there is no need to use a `csi` prefix. Every volume in this file is a csi volume.
May I suggest `logsDirVolumeName = "logs-dir"`